mdb-bi-connector/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:stretch
+MAINTAINER Matthias Granberry <matthias.granberry@youearnedit.com>
```

Comment:
the docker docs say this is deprecated. also, can this be an alias email? don't want to jinx it, but just in case someone leaves.

Reply:
Removed it since we really don't need it. It seems like they deprecated it because it was pointless.